### DISCUSSION TITLE
can textual generate plots?

### DISCUSSION THREAD

**Discussion by qiangbo** [Textualize/textual#39] [General]
Hi, can textual generate plots like plotext (https://pypi.org/project/plotext/)?

**Comment by willmcgugan** [comment]
If it works with Rich it works with Textual :)

https://github.com/piccolomo/plotext/issues/26

**Reply by Sonalice454** [reply]
Hello

**Reply by LumenYoung** [reply]
> If it works with Rich it works with Textual :)
> 
> [piccolomo/plotext#26](https://github.com/piccolomo/plotext/issues/26)

Hi, willmcgugan. 

Thank you for this wonderful framework.

I'm new to building TUI apps and is not so familiar with these concepts. I see that plotext supports rich in the thread you mentioned, but I don't know if we can use the library directly to generate plots in textual apps. 

If yes, can you briefly describe how to do it? Or what should I explore if such borrowing is not possible. Thanks in advance.

**Comment by davetapley** [comment]
More docs here: https://github.com/piccolomo/plotext/blob/master/readme/environments.md#rich
